service/controller/app/v1/resource/status/create.go

```diff
@@ -72,10 +69,10 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 	}
 
 	if !equals(desiredStatus, key.AppStatus(cr)) {
-		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("setting status for app %#q in namespace %#q", name, r.chartNamespace))
+		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("setting status for app %#q in namespace %#q", cr.Name, cr.Namespace))
```

Comment:
This was wrong. The chart namespace is `giantswarm` but the app CR is in the cluster namespace.